process/smartContract/process.go

```diff
@@ -690,6 +735,11 @@ func (sc *scProcessor) processVMOutput(
 
 	scrTxs = append(scrTxs, scrRefund)
 
+	err = sc.saveAccounts(acntSnd, nil)
```

Comment:
after these modifications, I do not think we need reloadAccount functions.

Reply:
use sc.accounts.SaveAccount

Reply:
Reload is still necessary. Refactored.